extension/awsproxy/factory.go

```diff
@@ -22,6 +22,7 @@ import (
 	"go.opentelemetry.io/collector/config/configtls"
 	"go.opentelemetry.io/collector/extension"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/extension/awsproxy/internal/metadata"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/proxy"
 )
 
```

Comment:
Can use `metadata.Type` as well instead of `typeStr`

Reply:
done